tests/phpt/declare/01_strict_types.phpt

```diff
@@ -0,0 +1,4 @@
+<?php
```

Comment:
Is it okay, that some files have the '.phpt' extension and don't have the '@ok' tag?

Reply:
I use `php` with + ok/fail tag for everything if it's possible.
`declare` can't be combined with tags as it will lead to a parse error in both KPHP and PHP.

Reply:
Files with `.phpt` extension implicitly have `@ok` tag - this is a hack for zend tests..
But for me, It's very confusing: some cases use `.php.inc`, but others `.phpt`, and from my opinion we shouldn't abuse this hack for our tests. I suggest remove all `.phpt` tests from `tests/phpt/` dir and use one style with `.php.inc`.

Reply:
Replaced `phpt` with `@ok` + require.